app/assets/javascripts/AppSearch.elm

```diff
@@ -75,7 +75,7 @@ update s msg model =
                                 Cmd.none
                     in
                         { model | results = Just results }
-                            ! (loadMore :: Map.fitContent :: addFacilities ++ [ Map.clearFacilityMarkers ])
+                            ! (loadMore :: Map.fitContent :: addFacilities :: [ Map.clearFacilityMarkers ])
```

Comment:
Why would we need a clearFacilityMarkers if addFacilities is actually a reset on those?